Userland/Libraries/LibCore/System.cpp

```diff
@@ -221,4 +221,12 @@ ErrorOr<void> chmod(StringView pathname, mode_t mode)
     return {};
 }
 
+ErrorOr<void> stat(StringView path, struct stat* statbuf)
```

Comment:
We already have this syscall, with a slightly more convenient API :^)

```cpp
ErrorOr<struct stat> stat(StringView path);
```

Reply:
Nice, updated :)